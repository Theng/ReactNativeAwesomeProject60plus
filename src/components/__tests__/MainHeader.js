import React from 'react';
import MainHeader from '../MainHeader';
import {render, cleanup} from 'react-native-testing-library';

afterEach(cleanup);

describe('<MainHeader />', () => {
  it('should match snapshot', () => {
    const rendered = render(<MainHeader />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});