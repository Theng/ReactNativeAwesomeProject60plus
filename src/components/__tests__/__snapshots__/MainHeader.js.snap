// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainHeader /> should match snapshot 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FAFAFA",
      "borderBottomColor": "#f2f2f2",
      "borderBottomWidth": 0.5,
      "elevation": 2,
      "flexDirection": "row",
      "height": 68,
      "justifyContent": "space-between",
      "paddingHorizontal": 0,
      "paddingTop": 20,
      "shadowOpacity": 1,
    }
  }
>
  <Image
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": 68,
          "width": undefined,
        },
        undefined,
      ]
    }
    testID="headerContainer"
  />
  <View
    style={
      Object {
        "alignItems": "flex-start",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
          "minWidth": 48,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#3E3E3E",
              "fontSize": 28,
            },
            null,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "flex": 3,
        "paddingHorizontal": 15,
      }
    }
  >
    <Text
      numberOfLines={1}
      style={
        Object {
          "color": "#000",
        }
      }
      text="MY TITLE"
      theme={
        Object {
          "colors": Object {
            "disabled": "hsl(208, 8%, 90%)",
            "divider": "#bcbbc1",
            "error": "#ff190c",
            "grey0": "#393e42",
            "grey1": "#43484d",
            "grey2": "#5e6977",
            "grey3": "#86939e",
            "grey4": "#bdc6cf",
            "grey5": "#e1e8ee",
            "greyOutline": "#bbb",
            "platform": Object {
              "android": Object {
                "error": "#f44336",
                "primary": "#2196f3",
                "secondary": "#9C27B0",
                "success": "#4caf50",
                "warning": "#ffeb3b",
              },
              "ios": Object {
                "error": "#ff3b30",
                "primary": "#007aff",
                "secondary": "#5856d6",
                "success": "#4cd964",
                "warning": "#ffcc00",
              },
            },
            "primary": "#2089dc",
            "searchBg": "#303337",
            "secondary": "#8F0CE8",
            "success": "#52c41a",
            "warning": "#faad14",
          },
        }
      }
    >
      MY TITLE
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-end",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
          "minWidth": 48,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#3E3E3E",
              "fontSize": 28,
            },
            null,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
